### ISSUE TITLE
Peak alignment

### ISSUE THREAD

**Issue by octavianparascinet** [OpenChrom/openchrom#168]
Hello! How can I do a peak alignment between up to 40 different samples? I saw there's an overlay view for the chromatograms, and there's also an alignment toolbar option. I press some of the alignment functions but it seems to me that it doesn't do nothing when I look back to the overlay perspective. If you could give me an example it'll work.

**Comment by Mailaender** [comment]
Take a look at https://lablicate.com/marketplace/extensions/chromident but it does not support alignment by warping the chromatogram, it only matches the peaks regardless of their retention time.

**Comment by octavianparascinet** [comment]
Ok, I understand. Therefore every time I'll be doing the alignment of the chromatograms there will be no stretching nor shrinkage of the different peaks. The function will only be detecting same peaks with similar retention times from the different chromatograms. But after alignment and detection of peaks from each chromatogram, will the identification function let me recognize automatically the same peaks as the same compound?

**Comment by Mailaender** [comment]
Yes, it matches the peaks from each chromatograms by mass spectra (even if the compound names differ slightly). See https://youtu.be/Hz6ti-Ed6LE

**Comment by octavianparascinet** [comment]
That's the problem. We only have the chromatograms because we just did the HPLC method on our samples. In that case it cannot work?

**Comment by Mailaender** [comment]
In that case, it is the wrong tool. Sorry.

**Comment by Mailaender** [comment]
We added peak alignment to https://lablicate.com/marketplace/extensions/powertools which is a commercial add-on for OpenChrom.
